tests/common/plugins/conditional_mark/tests_mark_conditions_202012.yaml

```diff
@@ -4,39 +4,39 @@
 ##############################################################
 
 dualtor/test_ipinip.py::test_decap_standby_tor:
-  xfail:
-    reason: "test issue or image issue, to be RCA'ed"
+  skip:
+    reason: "dual tor mock test only support on 7050cx3"
     conditions:
       - "release in ['202012']"
-      - "'dualtor' not in topo_name"
+      - "('dualtor' in topo_name) or ('7050cx3' not in platform.lower())"
```

Comment:
why are dual tor mock test only applicable for 7050cx3? What about 7260? @theasianpianist, any comments

Reply:
@neethajohn  @theasianpianist @lolyu 
add 7260 to the unskip, since we got dual tor testbeds, not sure still we need to maintain this mock cases? Those cases are flaky.